ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -130,160 +130,143 @@ export const AnacondaWizard = ({ dispatch, isBootIso, osRelease, storageData, lo
             data: {
                 deviceData: storageData.devices,
                 diskSelection: storageData.diskSelection,
-                requests: storageData.partitioning ? storageData.partitioning.requests : null,
```

Comment:
Thanks for shorting alphabetically the properties here and in the imports, but please do that in seperate commit if possible for code readability purposes.